c/test/legacy/cpp/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@ swift_add_test(test-libsbp-cpp-legacy
 swift_set_language_standards(test-libsbp-cpp-legacy)
 swift_set_compile_options(test-libsbp-cpp-legacy ADD -Wno-error=array-bounds)
 
-file(COPY sbp_data/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY sbp_data/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/c/test/legacy/cpp/sbp_data/)
```

Comment:
Bazel copies all files into a sandbox for building and execution. Data file dependencies are declared using the `data` attribute and are copied into the sandbox with the same path they have relative to the project root. This makes it so that the cmake version of the tests and the bazel version of the test have the files in the same place at runtime.